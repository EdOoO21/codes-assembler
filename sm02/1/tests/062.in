-646 -867 987 668 -513 827 -49 -366 838 373 -272 -650 40 -617 -670 -985 968 -467 304 -649 771 -749 -939 -33 -36 62 -962 587 778 -299 949 208 323 -672 416 -565 471 -839 -332 117 -83 152 -961 263 -186 916 923 -373 837 -299 -258 -52 844 937 -969 -371 770 -406 244 -127 -516 412 248 -428 -601 -646 226 498 -938 177 167 186 -591
