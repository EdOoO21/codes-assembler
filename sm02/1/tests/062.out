-591
186
167
177
-938
498
226
-646
-601
-428
248
412
-516
-127
244
-406
770
-371
-969
937
844
-52
-258
-299
837
-373
923
916
-186
263
-961
152
-83
117
-332
-839
471
-565
416
-672
323
208
949
-299
778
587
-962
62
-36
-33
-939
-749
771
-649
304
-467
968
-985
-670
-617
40
-650
-272
373
838
-366
-49
827
-513
668
987
-867
-646
